src/components/FAQ/FAQ.tsx

```diff
@@ -0,0 +1,67 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React, { ComponentPropsWithoutRef, ReactNode, useState } from 'react';
+import { FiChevronDown, FiChevronUp, FiMinus, FiPlus } from 'react-icons/fi';
+
+interface IFAQ extends ComponentPropsWithoutRef<'div'> {
+  children?: ReactNode;
+  title: string;
+  customClass?: string;
+}
+
+interface IFAQCategory extends ComponentPropsWithoutRef<'div'> {
+  categoryTitle: string;
+}
+
+export function FAQ(props: IFAQ) {
+  const { title, children, customClass } = props;
+  const [extend, setExtend] = useState(false);
+
+  return (
+    <div
+      className={`rounded-lg text-neutral border border-tertiary bg-primary-30 opactity-30 w-full ${customClass}`}
+      data-testid="faq"
+    >
+      {/* Header*/}
+      <div
+        className="flex items-center justify-between p-5 hover:cursor-pointer"
```

Comment:
```suggestion
        className="flex items-center justify-between p-5 cursor-pointer"
```

`cursor-pointer` automatically change the cursor on `hover`. So you can try that 🤔 😉